### PR TITLE
[zh-cn] Localize generated/kubectl_api-resources

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/kubectl/generated/kubectl_api-resources/_index.md

### CODE REVIEW

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_api-resources/_index.md [hunk 1]
```diff
@@ -0,0 +1,647 @@
...
+# Print the supported API resources sorted by a column
+# Print the supported namespaced resources
+# Print the supported non-namespaced resources
+# Print the supported API resources with a specific APIGroup
+-->
+```
+  # 打印服务器支持的 API 资源
+  kubectl api-resources
+  
+  # 打印支持的 API 资源及更多信息
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_api-resources/_index.md:41]:
```suggestion
  # 打印支持的 API 资源，但包含更多信息
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_api-resources/_index.md [hunk 2]
```diff
@@ -0,0 +1,647 @@
...
+<tr>
+<td colspan="2">--api-group string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<p>
+<!--
+Limit to resources in the specified API group.
+-->
+限制到指定 API 组中的资源。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_api-resources/_index.md:75]:
```suggestion
限制为指定 API 组中的资源。
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_api-resources/_index.md [hunk 3]
```diff
@@ -0,0 +1,647 @@
...
+<tr>
+<td colspan="2">--cached</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<p>
+<!--
+Use the cached list of resources if available.
+-->
+如果可用，请使用缓存的资源列表。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_api-resources/_index.md:89]:
```suggestion
如果可用，将使用缓存的资源列表。
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_api-resources/_index.md [hunk 4]
```diff
@@ -0,0 +1,647 @@
...
+<tr>
+<td colspan="2">--namespaced&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<!--Default: true-->默认值：true</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<p>
+<!--
+If false, non-namespaced resources will be returned, otherwise returning namespaced resources by default.
+-->
+如果为false，则返回非命名空间级别的资源，否则默认返回命名空间级别的资源。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_api-resources/_index.md:131]:
```suggestion
如果为false，则返回非命名空间作用域的资源，否则默认返回命名空间作用域的资源。
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_api-resources/_index.md [hunk 5]
```diff
@@ -0,0 +1,647 @@
...
+
+<tr>
+<td colspan="2">--cluster string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>
+<!--
+The name of the kubeconfig cluster to use
+-->
+kubeconfig 中要使用的集群的名称。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_api-resources/_index.md:318]:
```suggestion
要使用的 kubeconfig 中集群的名称。
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_api-resources/_index.md [hunk 6]
```diff
@@ -0,0 +1,647 @@
...
+
+<tr>
+<td colspan="2">--context string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>
+<!--
+The name of the kubeconfig context to use
+-->
+kubeconfig 要使用的上下文的名称。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_api-resources/_index.md:330]:
```suggestion
要使用的 kubeconfig 上下文的名称。
```

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**